packages/gatsby/src/commands/build-html.ts

```diff
@@ -1,18 +1,21 @@
-/* @flow */
-const Promise = require(`bluebird`)
-const webpack = require(`webpack`)
-const fs = require(`fs-extra`)
-// const convertHrtime = require(`convert-hrtime`)
-const { chunk } = require(`lodash`)
-const webpackConfig = require(`../utils/webpack.config`)
-const reporter = require(`gatsby-cli/lib/reporter`)
-const { createErrorFromString } = require(`gatsby-cli/lib/reporter/errors`)
-const telemetry = require(`gatsby-telemetry`)
+import Bluebird from "bluebird"
```

Comment:
Can't this just be imported as `Promise`?

Reply:
Typescript was throwing an error that we were overriding a global Promise. So I think we'll have to keep doing this

Reply:
Yup. I think i swapped all the regular promises out below for async/await